### PR TITLE
Fix resolving unicode fields

### PR DESCRIPTION
**Pull request by thomasst**

### CODE REVIEW

File: pybars/_compiler.py [hunk 1]
```diff
@@ -180,8 +180,14 @@ def escape(something, _escape_re=_escape_re, substitute=substitute):
 
 
 def pick(context, name, default=None):
-    if isinstance(name, str) and hasattr(context, name):
-        return getattr(context, name)
+    if isinstance(name, basestring):
+        try:
+            exists = hasattr(context, name)
+        except UnicodeEncodeError:
+            pass
```
Comment (wojcikstefan) [inline pybars/_compiler.py]: Could use a comment when this happens (unicode chars in `name` + Py2 only)

**Comment by wojcikstefan** [comment]
Does this let you use unicode context vars everywhere (e.g. conditionals, helper arguments, etc.)? If so, it could use a few extra unit tests.

**Comment by thomasst** [comment]
Only tested with looking up fields. Upstream PR: https://github.com/wbond/pybars3/pull/43

**Comment by isaacdd** [comment]
This is fixed now upstream with version 0.9.5. Thanks for the contribution!

**Pull request merged**